assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/formDecorators/utilities.js

```diff
@@ -0,0 +1,21 @@
+import path from 'ramda/src/path';
+import { amountsMatch } from '../../../../../application/utilities/money';
+export const isEqual = (value, prev) => prev === null || prev === undefined || value === prev;
+export const boolsEqual = (value, prev) => prev === null || prev === undefined || !!value === !!prev;
```

Comment:
what about Boolean(value) and Boolean(prev) ?
***
I think I mentioned this article before in this context https://www.sitepoint.com/javascript-double-negation-trick-trouble/

Reply:
ya missed that... will change to `Boolean(value)` thnx